vuepress/docs/next/docs/consume/identity-management.md

```diff
@@ -1,53 +1,66 @@
 # Entando Identity Management -- Keycloak
 
-Entando Identity Management is powered by Keycloak. The Keycloak instance used for your Entando apps
-can be [externally installed](../../tutorials/devops/external-id-management) or you can use a dedicated instance on a per application basis. The sections below details the architecture and documentation required to customize your Keycloak instance.
+Entando Identity Management is based on open source Keycloak. Entando Applications rely on a Keycloak instance that is either [externally installed](../../tutorials/devops/external-id-management) or specific to an application. The architecture and requirements to customize your Keycloak instance are described below.
 
 ## Logging into your Keycloak Instance
 
-In an Entando deployment Keycloak is protected by a Secret deployed in your Kubernetes instance. To get the default admin credentials you can query Kubernetes for the secret with this command:
+Keycloak is protected by a Secret deployed to your Entando Kubernetes instance. You can query Kubernetes for the Secret's default admin credentials, substituting your environment's namespace and Secret name:
 
 ```
-kubectl get secret quickstart-kc-admin-secret -n entando -o go-template="{{println}}Username: {{.data.username | base64decode}}{{println}}Password: {{.data.password | base64decode}}{{println}}{{println}}"
+ent kubectl get secret quickstart-kc-admin-secret -n entando -o go-template="{{println}}Username: {{.data.username | base64decode}}{{println}}Password: {{.data.password | base64decode}}{{println}}{{println}}"
 ```
 
-Update the namespace and secret name to match your environment. If you're not sure of the secret name you can run
+ To find the Secret name, run
 ```
 kubectl get secrets -n <namespace>
 ```
-And search for the secret that ends in `kc-admin-secret`
+and search for the Secret that ends in `kc-admin-secret`.
 
 ## Authentication
-In Entando 6 all authentication goes through Keycloak. This ensures that a micro frontend can call a microservice with a token that is available on the client.
+Beginning with Entando 6, all authentication is powered by Keycloak. This ensures that a micro frontend can call a microservice with a token available to the client.
 
 ![Init Containers Screenshot](./img/keycloak-arch-high-level.png)
 
-By using Keycloak as a central point of authentication the Entando architecture is able to provide a single unified view of identity to the entire architecture. With this architecture Entando becomes more portable and can be integrated into other IDPs without changes to the source. Keycloak acts as an  abstraction to the underlying IDP.
+The Entando architecture implements Keycloak as a central point of authentication. A single, unified view of identity increases Entando's portability. Entando can also integrate into other IDPs without modifying the source, because Keycloak acts as an abstraction to the underlying IDP.
```

Comment:
Does the Entando architecture implement Keycloak or does it provide a unified view of id... >> the original verbiage gave a clearer view. This seemed more confusing. What is IDP?

Reply:
the architecture implements Keycloak to provide a unified view. it should be obvious that central point of authentication provides a single, unified view ?? will rework...